rfcs/text/0000-deprecate-log-original.md

```diff
@@ -0,0 +1,106 @@
+# 0000: Name of RFC
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+This RFC supersedes issue #841 which implies breaking changes therefore the RFC Process is indicated
+
+The request is to consolidate `log.original` and `event.original` by removing `log.original`, since these are almost equivalent in nature. (One) justification for preserving `event.original` is that not all events are logs.
```

Comment:
Would be worth including that once `log.original` is removed, `event.original` will be the sole field intended to capture the original untouched event.

Reply:
applied